src/main/resources/reference.conf

```diff
@@ -31,32 +31,36 @@ jicofo {
 
     // RTP header extensions
     rtp-extensions {
+      audio-level {
+        enabled=true
+        id=1
+      }
```

Comment:
Turning off audio levels will break a bunch of bridge behavior -- is this just a case of "you get what you ask for"? Is there a use case for turning them off, other than symmetry among the extensions?

Reply:
I think "you get what you ask for" describes it well.